src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -106,6 +106,9 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
   @Input()
   autoSave = true;
 
+  @Input()
+  searchableObjects = false;
```

Comment:
Please add a documentation what this input variable is for.

Reply:
Done :)